File: app/src/main/java/com/bilulo/androidtest04/data/model/response/StatementsResponse.java
package com.bilulo.androidtest04.data.model.response;

import com.bilulo.androidtest04.data.model.ErrorModel;
import com.bilulo.androidtest04.data.model.StatementModel;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class StatementsResponse {
    @SerializedName("statementList")
    private List<StatementModel> statementModelList;
    @SerializedName("error")
    private ErrorModel errorModel;

    public List<StatementModel> getStatementModelList() {
        return statementModelList;
    }

    public void setStatementModelList(List<StatementModel> statementModelList) {
        this.statementModelList = statementModelList;
    }

    public ErrorModel getErrorModel() {
        return errorModel;
    }

    public void setErrorModel(ErrorModel errorModel) {
        this.errorModel = errorModel;
    }
}

File: app/src/main/java/br/com/flaviokreis/santanderv2/data/response/StatementsResponse.kt
package br.com.flaviokreis.santanderv2.data.response

import br.com.flaviokreis.santanderv2.data.model.Error
import br.com.flaviokreis.santanderv2.data.model.Statement

data class StatementsResponse(
    val statementList: List<Statement>,
    val error: Error?
)